testsuite/src/it/java/org/graalvm/tests/integration/utils/WhitelistLogLines.java

```diff
@@ -130,6 +135,59 @@ public Pattern[] get(boolean inContainer) {
             };
         }
     },
+    QUARKUS_MP_ORM_DBS_AWT {
+        @Override
+        public Pattern[] get(boolean inContainer) {
+            final List<Pattern> p = new ArrayList<>();
+            // Our config
+            p.add(Pattern.compile(".*Unrecognized configuration key \"quarkus.version\".*"));
+            // Testcontainers might not need it, depends on your system.
+            p.add(Pattern.compile(".*Attempted to read Testcontainers configuration file.*"));
+            p.add(Pattern.compile(".*does not support the reuse of containers.*"));
+            // GC warning thrown in GraalVM >= 22.0 under constraint environment (e.g. CI)
+            // see https://github.com/Karm/mandrel-integration-tests/issues/68
+            p.add(Pattern.compile(".*GC warning: [0-9.]+s spent in [0-9]+ GCs during the last stage, taking up [0-9]+.[0-9]+% of the time.*"));
+            // JUnit output
+            p.add(Pattern.compile(".* Failures: 0, Errors: 0,.*"));
+            // RestEasy intermittently
+            p.add(Pattern.compile(".*Closing a class org.jboss.resteasy.client.*"));
+            p.add(Pattern.compile(".*Please close clients yourself.*"));
```

Comment:
I wonder if this is a quarkus problem or a test issue that needs handling there. Either way, further investigation seems waranted. See https://github.com/quarkusio/quarkus/issues/10813#issuecomment-1699360692

Reply:
I thought it was me, not bothering with the rest client here:

https://github.com/Karm/mandrel-integration-tests/blob/master/apps/quarkus-full-microprofile/src/main/java/com/example/quarkus/secure/TestSecureController.java#L45

So I did treat the closable this time around:

https://github.com/Karm/mandrel-integration-tests/pull/254/files#diff-e855f86a0889eaeb6ad4c059c62b3d11064d2a7e2d57a44d6df6d70e6c9507f9R49

...to no avail. IMHO it's something else. Like CDI scope being wrong? Perhaps it's expected to be just RequestScoped and by minimizing boilerplate and jamming everything together in App scoped in the test case, we are triggering it. It's not specific to native though.